editor/src/components/editor/actions/actions.ts

```diff
@@ -965,6 +965,7 @@ export function restoreDerivedState(history: StateHistory): DerivedState {
 
   return {
     navigatorTargets: poppedDerived.navigatorTargets,
+    visibleNavigatorTargets: poppedDerived.visibleNavigatorTargets,
```

Comment:
I raised my eyebrow, and then I realized collapsedViews is in the undo history. eek!
 `collapsedViews: poppedEditor.navigator.collapsedViews` 

now that I noticed it, I _think_ we should change this to not be undoable, unless I am missing something?

Reply:
I guess it's only an issue if the action triggers a checkpoint in the undo history right?